AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -380,9 +380,16 @@ final class HiddenCoreDataMeasurementStreamStorage: MeasurementStreamStorageCont
                     long: note.long,
                     number: Int(note.number))
     }
+    
+    private func newSessionEntity() -> SessionEntity {
```

Comment:
I thought we were supposed to remember the state only for followed sessions, based on the trello ticket title. Was that changed? If we should do it only for followed sessions, maybe we should add UIState to session only on follow button click?

Reply:
I missed it totally. I will talk to Michael, thanks!